CHANGELOG.md

```diff
@@ -21,6 +28,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Fixed
 - Fix issue with notebook ip address in config (#50)
 
+### Removed
+- File logging. All logs will now be written to stdout/stderr streams. (#47)
```

Comment:
This item had been released in v1.0.1 but wasn't included in the changelog for that version.